packages/image_picker/image_picker/example/lib/main.dart

```diff
@@ -174,6 +194,56 @@ class _MyHomePageState extends State<MyHomePage> {
     }
   }
 
+  Widget _previewMultiImages() {
+    final Text? retrieveError = _getRetrieveErrorWidget();
+    if (retrieveError != null) {
+      return retrieveError;
+    }
+    if (_imageFileList != null) {
+      if (kIsWeb) {
+        // Why network?
+        // See https://pub.dev/packages/image_picker#getting-ready-for-the-web-platform
+        return ListView.builder(
+          key: UniqueKey(),
+          itemBuilder: (context, index) {
+            return Image.network(_imageFileList![index].path);
+          },
+          itemCount: _imageFileList!.length,
+        );
+      } else {
+        return Semantics(
+            child: ListView.builder(
+              key: UniqueKey(),
+              itemBuilder: (context, index) {
+                return Image.file(File(_imageFileList![index].path));
+              },
+              itemCount: _imageFileList!.length,
+            ),
+            label: 'image_picker_example_picked_images');
+      }
```

Comment:
I think doing the if (kIsWeb) within the builder is cleaner and has less duplicate code. For example:
```suggestion
      return Semantics(
          child: ListView.builder(
            key: UniqueKey(),
            itemBuilder: (context, index) {
              if (kIsWeb) {
                // Why network?
                // See https://pub.dev/packages/image_picker#getting-ready-for-the-web-platform
                return Image.network(_imageFileList![index].path);
              }
              return Image.file(File(_imageFileList![index].path));
            },
            itemCount: _imageFileList!.length,
          ),
          label: 'image_picker_example_picked_images');
```

Reply:
I thought about the `kIsWeb` part and I think we can remove it completely since `getMultiImage` doesn't support web yet. What do you think?